metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/impl/StormStatusServiceImpl.java

```diff
@@ -17,10 +17,7 @@
  */
 package org.apache.metron.rest.service.impl;
 
-import org.apache.metron.rest.model.TopologyResponse;
-import org.apache.metron.rest.model.TopologyStatus;
-import org.apache.metron.rest.model.TopologyStatusCode;
-import org.apache.metron.rest.model.TopologySummary;
+import org.apache.metron.rest.model.*;
```

Comment:
Thanks for checking me on this @mattf-horton.  I was under the impression we already voted to switch our coding standard to Google's based on the [Code Style](http://mail-archives.apache.org/mod_mbox/metron-dev/201705.mbox/%3CCADu=U3A23q-aehhim6R0AceZapAJ9QJvG8hnurhR2hkDtvXNLw@mail.gmail.com%3E) discussion on the dev list back in the beginning of May.  There is also a pending [PR](https://github.com/apache/metron/pull/577) that sets up Checkstyle with the Google Code Style.

Was this not an official vote or was the wiki just not updated?  If we did switch, [Google Coding Style](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements) does not allow wildcard imports.

This is probably one of those topics with no clear correct answer, similar to spaces vs tabs.  I personally am not a fan of wildcard statements because I prefer to see exactly what is being imported.